presentation/home/src/main/java/com/dhc/home/HomeViewModel.kt

```diff
@@ -274,4 +278,23 @@ class HomeViewModel @Inject constructor(
             )
         }
     }
+
+    fun finishTodayMission() {
+        viewModelScope.launch {
+            val userId = userRepository.getUserId().firstOrNull() ?: return@launch
+            dhcRepository.requestFinishTodayMissions(
+                userId = userId,
+                EndTodayMissionRequest(
+                    date = todayStringFormat
+                )
```

Comment:
nit) 별건 아니지만 일관되면 더 보기 좋을 것 같아서 ...
```suggestion
                userId = userId,
                endTodayMissionRequest = EndTodayMissionRequest(
                    date = todayStringFormat
                )
```

Reply:
oops..my mistake~